src/framework/preferences/js/Panels.js

```diff
@@ -683,20 +706,33 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
                 }
             }
         },
-        protoTree: {
-            label: {messagekey: "textSizeLabel"},
-            smallIcon: {messagekey: "textSizeSmallIcon"},
-            largeIcon: {messagekey: "textSizeLargeIcon"},
-            multiplier: {messagekey: "multiplier"},
-            textSizeDescr: {messagekey: "textSizeDescr"}
-        },
         sliderOptions: {
             orientation: "horizontal",
             step: 0.1,
             range: "min"
         }
     });
 
+    fluid.defaults("fluid.prefs.textSize.textfieldButtons", {
```

Comment:
Noticed you've moved the rendering of the label and description to the parent component so this comment is not valid.